mne/baseline.py

```diff
@@ -54,7 +54,7 @@ def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
     copy : bool
         Whether to return a new instance or modify in place.
     verbose : bool, str, int, or None
-        If not None, override default verbose level (see mne.verbose).
+        If not None, override default verbose level (see :func:`mne.verbose`).
```

Comment:
Something just occurred to me. We could use a trick like what @wmvanvliet did with `function_doc_to_method_doc` to automatically document `verbose`. We know it is always going to be the last argument in the argument list...